src/components/atoms/Combo/Combo.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * combo (label / drop-down) component
+ */
+export function Combo(props) {
+  const {
+    customClass,
+    customLabelClass,
+    customSelClass,
+    id,
+    lblText,
+    required,
+    reqText,
+    options,
+    onChange,
+  } = props;
+
+  return (
+    <div
+      className={`p-4 w-full flex justify-center content-center ${customClass} ? customClass : '' `}
+    >
+      <div>
+        <label
```

Comment:
would probably split this into another component so that it can be reused and its cleaner